test/e2e/service_bootstrap.py

```diff
@@ -16,12 +16,65 @@
 
 from acktest.bootstrapping import Resources, BootstrapFailureException
 from e2e import bootstrap_directory
+from acktest.bootstrapping.s3 import Bucket
 from e2e.bootstrap_resources import BootstrapResources
+from acktest.resources import random_suffix_name
+from e2e.common import TRAIL_NAME
+from acktest.aws.identity import get_region, get_account_id
+
+BUCKET_NAME = random_suffix_name("trail-bucket-", 24)
```

Comment:
I don't think this should be a global. Maybe use a `.replace` inside the `service_bootstrap` method.